pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1734,15 +1734,53 @@ func buildTracingConfig(config *meshconfig.ProxyConfig) *hcm.HttpConnectionManag
 					Value: config.Tracing.MaxPathTagLength,
 				}
 		}
-
-		if len(config.Tracing.CustomTags) != 0 {
-			tracingCfg.CustomTags = buildCustomTags(config.Tracing.CustomTags)
-		}
+		tracingCfg.CustomTags = buildCustomTags(config.Tracing.CustomTags)
 	}
 
 	return tracingCfg
 }
 
+func defaultTags() []*tracing.CustomTag {
```

Comment:
Do we have any doc and/or test asserting we are actually setting these env vars? I am just a bit worried about someone like me accidentally removing them since they are "not used"

Reply:
Looks like we do have a test, didn't finish the PR :slightly_smiling_face: